rate-providers/registry.json

```diff
@@ -1,1007 +1,1328 @@
+      "asset": "0xa2E3356610840701BDf5611a53974510Ae27E2e1",
+      "name": "wBETH BinanceBeaconEthRateProvider",
+      "summary": "safe",
+      "review": "./LegacyReview.md",
+      "warnings": ["legacy"],
+      "factory": "",
+      "upgradeableComponents": []
+    },
+    "0x12589A727aeFAc3fbE5025F890f1CB97c269BEc2": {
+      "asset": "0x4Bc3263Eb5bb2Ef7Ad9aB6FB68be80E43b43801F",
+      "name": "Bitfrost vETHRateProvider",
+      "summary": "safe",
+      "review": "./LegacyReview.md",
+      "warnings": ["legacy"],
+      "factory": "",
+      "upgradeableComponents": []
+    },
+    "0x5F0A29e479744DcA0D3d912f87F1a6E3237A55D3": {
+      "asset": "0x0Ae38f7E10A43B5b2fB064B42a2f4514cbA909ef",
+      "name": "unshETHRateProvider",
+      "summary": "safe",
+      "review": "./LegacyReview.md",
+      "warnings": ["legacy"],
+      "factory": "",
+      "upgradeableComponents": []
+    }
+  },
+  "fantom": {
+    "0x629d4c27057915e59dd94bca8d48c6d80735b521": {
+      "asset": "0xd7028092c830b5c8fce061af2e593413ebbc1fc1",
+      "name": "sFTMx Rateprovider",
+      "summary": "safe",
+      "review": "./LegacyReview.md",
+      "warnings": ["legacy"],
+      "factory": "",
+      "upgradeableComponents": []
+    }
+  },
+  "gnosis": {
+    "0x89C80A4540A00b5270347E02e2E144c71da2EceD": {
+      "asset": "0xaf204776c7245bF4147c2612BF6e5972Ee483701",
+      "name": "ERC4626RateProvider",
+      "summary": "safe",
+      "review": "./SavingsDAIRateProviderGnosis.md",
+      "warnings": ["donation", "only18decimals"],
+      "factory": "",
+      "upgradeableComponents": []
+    },
+    "0xff315299C4d3FB984b67e31F028724b6a9aEb077": {
+      "asset": "0x004626A008B1aCdC4c74ab51644093b155e59A23",
+      "name": "ERC4626RateProvider",
+      "summary": "safe",
+      "review": "./stAgEurRateProvider.md",
+      "warnings": ["donation", "only18decimals"],
+      "factory": "",
+      "upgradeableComponents": [
+        {
+          "entrypoint": "0x004626A008B1aCdC4c74ab51644093b155e59A23",
+          "implementationReviewed": "0x6C04c39B9E73aC91106D12F828e2E29Fd8ef1024"
+        },
+        {
+          "entrypoint": "0x4b1E2c2762667331Bc91648052F646d1b0d35984",
+          "implementationReviewed": "0x59153e939c5b4721543251ff3049Ea04c755373B"
+        }
+      ]
+    },
+    "0x09f9611FE9d24c6A518f656E925e3628A2ECDE3b": {
+      "asset": "0x6C76971f98945AE98dD7d4DFcA8711ebea946eA6",
+      "name": "wstETH Rate Provider",
+      "summary": "safe",
+      "review": "",
+      "warnings": ["chainlink"],
+      "factory": "0xE7511f6e5C593007eA8A7F52af4B066333765e03",
+      "upgradeableComponents": []
+    },
+    "0xE7511f6e5C593007eA8A7F52af4B066333765e03": {
+      "asset": "0xcB444e90D8198415266c6a2724b7900fb12FC56E",
+      "name": "EURe Rate Provider",
+      "summary": "safe",
+      "review": "./ChainlinkRateProvider.md",
+      "warnings": ["chainlink"],
+      "factory": "0xE7511f6e5C593007eA8A7F52af4B066333765e03",
```

Comment:
@Zen-Maxi Can you provide the address for the factory on gnosis. Looks like there is a hiccup here.

Reply:
@Zen-Maxi The Rps in question were deployed without a factory. I have removed the factory entries.